src/Microsoft.AspNet.Mvc.Core/ModelBinding/MutableObjectModelBinder.cs

```diff
@@ -374,11 +380,13 @@ internal static PropertyValidationInfo GetPropertyValidationInfo(ModelBindingCon
             return validationInfo;
         }
 
-        internal void ProcessDto(ModelBindingContext bindingContext, ComplexModelDto dto)
+        internal ModelValidationNode ProcessDto(
```

Comment:
while here, could you add an `// Internal for testing` comment for such methods in this class?